spec/controllers/assignments_controller_spec.rb

```diff
@@ -74,12 +74,13 @@
         end
 
         it 'gives errors' do
-          expect { submit }.to redirect_back
+          submit
+          expect(flash[:errors]).not_to be_empty
         end
 
-        it 'redirects back' do
+        it 'redirects' do
           submit
-          expect(flash[:errors]).not_to be_empty
+          expect(response).to have_http_status(:found)
```

Comment:
```ruby
          expect(response).to redirect_to(wherever_were_supposed_to_go_path)
```
Redirects have their own matcher that also checks the path that we're supposed to go